### ISSUE TITLE
[12] Fields invisible on tree view, not load value on form view

### ISSUE THREAD

**Issue by celm1990** [odoo/odoo#33592]
**Impacted versions:**
12, tested on runbot
**Steps to reproduce:**
When for any reason a field type many2many with widget="many2many_tags" is present on tree view as invisible="1", at open form view not load/show value.

Case of use: In the example I take invoice lines and field `invoice_line_tax_ids`, but I really have error on custom module, but issue is reproducible on any view with fields many2many, for this reason i create issue, on my opinion is error of framework.

@ged-odoo you is man for this type issue, sorry for ping 
Current behavior:

![invisible tree view](https://user-images.githubusercontent.com/7775116/58198904-f3468580-7c94-11e9-914e-8bf14e64cca1.gif)

Expected behavior:

Video/Screenshot link (optional):

**Comment by sswapnesh** [comment]
Hello @celm1990 
> but I really have error on custom module.

Is it happening even if you are not adding `many2many_tags` there? (and What is the use case for adding  many2many_tags on a invisible field ? )

I have checked and removing '`many2many_tags`' from tree with `invisible="1"` works well there. (At Least It can solve your Issues)

**Comment by celm1990** [comment]
> Hello @celm1990
> 
> > but I really have error on custom module.
> 
> Is it happening even if you are not adding `many2many_tags` there? (and What is the use case for adding many2many_tags on a invisible field ? )
> 
> I have checked and removing '`many2many_tags`' from tree with `invisible="1"` works well there. (At Least It can solve your Issues)

Of course, without widget `many2many_tags` worked fine, but think is error of framework with widget `many2many_tags`.

**Comment by pedrobaeza** [comment]
Coming to this through codetriage.com. Is this still happening?

**Comment by C3POdoo** [comment]
Dear @celm1990,

Thank you for your report but we are closing it due to inactivity.
We apology if we could not look at your request in time.
If your report still makes sense, don't hesitate to reopen a new one. We will try to check it as soon as possible.

_This is an automated message._
